nautobot/extras/migrations/0032_tag_content_types_data_migration.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.1.14 on 2022-03-19 23:21
+
+from django.db import migrations
+import nautobot.extras.utils
+
+
+def populate_existing_tags(app, schema):
+    Tag = app.get_model("extras", "Tag")
+    content_types = nautobot.extras.utils.TaggableClassesQuery().get_choices()
+
+    for tag in Tag.objects.filter(content_types__isnull=True):
+        for _, id in content_types:
+            tag.content_types.add(id)
+
```

Comment:
A reverse migration function here *might* be desirable?

Reply:
TODO: Add reverting data migration (remove many-to-many relationships).